apps/agentic-chat/package.json

```diff
@@ -34,7 +36,8 @@
     "@shapeshiftoss/caip": "8.15.0",
     "@tanstack/react-query": "^5.75.5",
     "viem": "^2.7.9",
-    "wagmi": "^2.15.2"
+    "wagmi": "^2.15.3",
+    "@wagmi/core": "^2.17.2"
```

Comment:
Note explicit `@wagmi/core` - needed to add this as transient deps (`@reown/appkit-adapter-wagmi` to be exact) are derp and don't explicitly have it in their package.json